src/components/ConnectorTiles.tsx

```diff
@@ -188,45 +187,38 @@ function ConnectorTiles({
                     .map((row, index) => (
                         <ConnectorCard
                             key={`connector-tile-${index}`}
-                            imageSrc={row.image}
-                            lastUpdate={row.updated_at}
-                            title={row.title}
                             docsUrl={row.connector_tags[0].documentation_url}
-                            entity={row.connector_tags[0].protocol}
-                            ctaCallback={() => primaryCtaClick(row)}
-                            description={row.detail}
+                            logo={<ConnectorCardLogo imageSrc={row.image} />}
+                            title={<ConnectorCardTitle title={row.title} />}
+                            details={
+                                <ConnectorCardDetails
+                                    description={row.detail}
+                                    lastUpdate={row.updated_at}
+                                />
+                            }
+                            cta={
+                                <ConnectorCardCTA
+                                    ctaCallback={() => primaryCtaClick(row)}
+                                    entity={row.connector_tags[0].protocol}
+                                />
+                            }
```

Comment:
Refactoring: passing components around instead of props now to let the "request connector" use the component